eloquent-js/chap14/chap14q3.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Eloquent JS Chapter 14 Q3</title>
+    <style>
+        body {
+            min-height: 200px;
+        }
+
+        img {
+            height: 30px;
+
+        }
+    </style>
+</head>
+<body>
+    <img src="img/cat.png" id="cat" style="position: absolute">
+    <img src="img/hat.png" id="hat" style="position: absolute">
+
+    <script>
+        let cat = document.querySelector("#cat");
+        let hat = document.querySelector("#hat");
+
+        let angle = 0;
+        let lastTime = null;
+        function animate(time) {
+            if (lastTime != null) angle += (time - lastTime) * 0.001;
+            lastTime = time;
+
+            const catTop = (Math.sin(angle) * 100 + 200);
+            const catLeft = (Math.cos(angle) * 200 + 230);
+
+            cat.style.top = catTop + "px";
+            cat.style.left = catLeft + "px";
+
+            hat.style.top = catTop + -40 + "px";
+            hat.style.left = catLeft + "px";
+            hat.style.transform = "scale(" + Math.sin(3 * angle) + ", 1)";
```

Comment:
This is a good opportunity to use template literal (and extract the new value to a variable fro readability)